shells/android/javatests/arcs/crdt/parcelables/ParcelableRawEntityTest.kt

```diff
@@ -0,0 +1,31 @@
+package arcs.crdt.parcelables
+
+import android.os.Parcel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import arcs.data.RawEntity
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ParcelableRawEntityTest {
+    @Test
+    fun parcelableRoundTrip_works() {
```

Comment:
It might be good to add a test with a RawEntity which contains RawEntity value(s) for its singleton/collection entry(-ies). (a test of nested rawentities)

Reply:
Done